src/compas_rhino/interop/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class COMPASRhinoInteropError(Exception):
```

Comment:
How about `ConversionError`? the class is already in the `compas_rhino` package, it's redundant to add that to the name as well.